### ISSUE TITLE
Static method to return Intent array

### ISSUE THREAD

**Issue by caefree** [airbnb/DeepLinkDispatch#63]
Currently, the static method needs to return a single Intent object

```
 public static Intent newIntent(Context context) {
        return new Intent(context, BlahActivity.class);
    }
```

Ideally, we'd like to be able to work with something like this:

```
    public static Intent[] newIntent(Context context) {
        return new Intent[] { newIntentOne(context), newIntentTwo(context) };
    }
```

**Comment by felipecsl** [comment]
Can you elaborate on the reason why you need that?
Currently, the returned Intent is used to launch the Activity along with the extra params from the deep link. 
If you return an array, how do you determine which Intent to use?

**Comment by caefree** [comment]
@felipecsl Since the deeplinks go straight into the desired Activity, sometimes we would like to have another Activity on the backstack before that.

You wouldn't need to choose which intent to fire.
There is a method in "Context" called startActivities which receive an Intent[] array.

http://developer.android.com/reference/android/content/Context.html#startActivities(android.content.Intent[])

**Comment by felipecsl** [comment]
Can you just use [`parentActivityName`](http://developer.android.com/training/implementing-navigation/ancestral.html#SpecifyParent) instead?

**Comment by caefree** [comment]
@felipecsl That would work for the up button, but not for the back button.

http://developer.android.com/training/implementing-navigation/temporal.html#SynthesizeBackStack --> they also mention the method "startActivities" here a bit further down below.

**Comment by felipecsl** [comment]
OK makes sense. we can add support for that tehn

**Comment by caefree** [comment]
@felipecsl that would be awesome, thanks!

**Comment by caefree** [comment]
@felipecsl any  updates on this ? Is it likely to getting picked up anytime soon?

**Comment by skptl** [comment]
It would be great to have such method, by that we can actually create an entire task stack.
@felipecsl Any plans for this?
Thanks.

**Comment by felipecsl** [comment]
We don't have bandwidth to address this right now, but please feel free to submit a PR!

**Comment by felipecsl** [comment]
Fixed by #120
